src/holdup/cli.py

```diff
@@ -237,6 +270,12 @@ def parse_value(value, proto):
             raise argparse.ArgumentTypeError('Invalid service spec %r. Port must be a number.' % value)
         port = int(port)
         return TcpCheck(host, port)
+    elif proto == 'postgresql':
+        user, password, host, port, dbname = value.strip('/').split(':', 4)
```

Comment:
I think we should have a different syntax. For tcp protocol it's very clear and simple to parse (`host:port`) but for postgres we have to many things. `postgres://user:password:host:port:dbname` is not really intuitive.

Fortunately psycopg2 supports DSNs (https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) so we can just use those and not do any parsing at all.

Parse with `psycopg2.extensions.make_dsn` and just pass its result to `psycopg2.connect`.